qcodes/instrument_drivers/rohde_schwarz/ZNB.py

```diff
@@ -382,9 +383,14 @@ class ZNB(VisaInstrument):
     CHANNEL_CLASS = ZNBChannel
 
 
-    def __init__(self, name: str, address: str, init_s_params: bool=True, **kwargs) -> None:
+    def __init__(self, name: str, address: str,
+                 init_s_params: bool=True, **kwargs) -> None:
 
-        super().__init__(name=name, address=address, **kwargs)
+        # NOTE(WilliamHPNielsen): there is an implicit assumption in the
+        # val_mappings of this driver that the terminator is '', i.e. that
+        # a '\n' is also returned
```

Comment:
This sounds like the original author did something wrong? :)

Reply:
I would be tempted to say yes, but perhaps he had some binary data transfer in mind when he did it? I am at least a little worried about breaking the trace acquisition if we change the termination.

Reply:
let's leave it like this)